src/core/evaluator.js

```diff
@@ -2399,40 +2587,10 @@ class PartialEvaluator {
             textState.textMatrix = textState.textLineMatrix.slice();
             break;
           case OPS.nextLine:
-            flushTextContentItem();
+            appendEOL();
             textState.carriageReturn();
             break;
           case OPS.setTextMatrix:
-            // Optimization to treat same line movement as advance.
-            advance = textState.calcTextLineMatrixAdvance(
```

Comment:
If the patch results in this method becoming unused, and you don't plan on using it for any (possibly) upcoming changes, please remove this code as well: https://github.com/mozilla/pdf.js/blob/c86e70ba08bcde6b2b510bbfd60dc5567ac5285b/src/core/evaluator.js#L3890-L3915

---

Also, it appears that this method has been unused for *years* and can also be removed: https://github.com/mozilla/pdf.js/blob/c86e70ba08bcde6b2b510bbfd60dc5567ac5285b/src/core/evaluator.js#L3917-L3928